library/src/main/java/com/bumptech/glide/request/RequestListener.java

```diff
@@ -68,18 +69,19 @@ boolean onLoadFailed(
    *
    * <p>For threading guarantees, see the class comment.
    *
-   * @param resource The resource that was loaded for the target.
-   * @param model The specific model that was used to load the image.
-   * @param target The target the model was loaded into.
-   * @param dataSource The {@link DataSource} the resource was loaded from.
+   * @param resource        The resource that was loaded for the target.
+   * @param model           The specific model that was used to load the image.
+   * @param target          The target the model was loaded into.
+   * @param dataSource      The {@link DataSource} the resource was loaded from.
    * @param isFirstResource {@code true} if this is the first resource to in this load to be loaded
-   *     into the target. For example when loading a thumbnail and a full-sized image, this will be
-   *     {@code true} for the first image to load and {@code false} for the second.
+   *                        into the target. For example when loading a thumbnail and a full-sized image, this will be
+   *                        {@code true} for the first image to load and {@code false} for the second.
    * @return {@code true} to prevent {@link Target#onResourceReady(Object, Transition)} from being
-   *     called on {@code target}, typically because the listener wants to update the {@code target}
-   *     or the object the {@code target} wraps itself or {@code false} to allow {@link
-   *     Target#onResourceReady(Object, Transition)} to be called on {@code target}.
+   * called on {@code target}, typically because the listener wants to update the {@code target}
+   * or the object the {@code target} wraps itself or {@code false} to allow {@link
+   * Target#onResourceReady(Object, Transition)} to be called on {@code target}.
    */
   boolean onResourceReady(
-      R resource, Object model, Target<R> target, DataSource dataSource, boolean isFirstResource);
```

Comment:
If these don't all fit on one line, please put each parameter on its own new line:

```
onResourceReady(
    R resource,
    Object model,
    ...
)
```
etc

Reply:
That's not my change. This is the only thing i did:

![image](https://user-images.githubusercontent.com/5357526/190845604-dd61ccae-51d3-430f-93b2-2cafcd832553.png)

Reply:
I'm not sure what to say, it shows up in the diff in Github, so it's in whatever's committed... Maybe you've fixed or reverted this already, try force pushing?

Reply:
@sjudd What's "force pushing"? Where do I do this here? I already pushed into my fork. I can push directly to master ? 
Please explain where do I do this.
Maybe I should remove my fork, and fork again and try again?